script/lib/l10n/crowdin/common.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import json
+import re
+import os
+import lxml.etree  # pylint: disable=import-error
+
+from lib.l10n.grd_utils import textify
+
+# pylint: disable=import-error
+from lib.l10n.crowdin.api_v2_client_wrapper import CrowdinClientWrapper
+# pylint: enable=import-error
+
+brave_project_id = 6  # Brave Core (Android+Chrome)
+
+# This module contains functionality common to both pulling down translations
+# from Crowdin and pushing source strings up to Crowdin.
+
+# Filenames that are fully handled by Crowdin (as opposed to files for which
+# we create overrides that are then handled by Crowdin).
+crowdin_handled_files = [
+    'android_brave_strings.xml',
+    'brave_generated_resources.xml',
+    'brave_components_strings.xml',
+    'brave_extension.json',
+]
+
+
+def should_use_crowdin_for_file(source_string_path, filename):
+    """ Determines if the given file should be handled by Crowdin locally"""
+    name = crowdin_name_from_filename(source_string_path, filename)
+    return name in crowdin_handled_files or name.startswith('greaselion_')
```

Comment:
I doubt we need to support the Greaselion translations any more. cc: @zenparsing

Reply:
Oh yeah, nice find. I removed some other greaselion code, but missed this.

Reply:
Fixed in [13140e2](https://github.com/brave/brave-core/pull/27005/commits/13140e28ea7cd53e4056067f06938b892d409c16)